scripts/metadata_comparison/metadata_comparison/comparer.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python3
+#
+# comparer.py
+#
+# Purpose: Compare performance metadata JSON files produced by Digester and produce result in CSV format
+#
+# Usage: python3 comparer.py [-h] [-v] [--json_paths JSONPATH [JSONPATH ...]] [--output_path OUTPUTPATH]
+#
+# Python Prereqs (at least, the ones which I needed to manually install... YMMV):
+#
+#   * pip3 install --upgrade pandas
+#
+
+import argparse
+import json
+import pandas
+from pathlib import Path
+import logging as log
+
+logger = log.getLogger('metadata_comparison.comparer')
+
+def set_log_verbosity(verbose):
+    if verbose:
+        log.basicConfig(format='[%(asctime)s] [%(name)s] %(message)s', level=log.INFO)
+    else:
+        log.basicConfig(format='[%(asctime)s] [%(name)s] %(message)s', level=log.WARNING)
+
+
+def read_json_files(*paths):
+    """
+    Produces list of tuples: [[path, json], [path, json], ...]
+    """
+    jsons = []
+    for path in paths:
+        jsons.append([path, json.load(open(path,))])
+
+    return jsons
+
```

Comment:
I don't think it changes the parsing too much, but I try to follow the python practice of having double newline after `def`s

Reply:
PEP8 wants that too and my IntelliJ seems to be a PEP8 enforcer